lib/pbench/server/database/alembic/versions/9df060db17de_tokens_store_expiration.py

```diff
@@ -0,0 +1,33 @@
+"""Tokens store expiration instead of creation.
+
+Revision ID: 9df060db17de
+Revises: fa12f45a2a5a
+Create Date: 2023-02-05 15:58:25.408754
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9df060db17de"
+down_revision = "fa12f45a2a5a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.alter_column(
+        "active_tokens", "created", nullable=False, new_column_name="expiration"
+    )
+    op.create_index(
+        op.f("ix_active_tokens_expiration"),
+        "active_tokens",
+        ["expiration"],
+        unique=False,
+    )
+
+
+def downgrade() -> None:
+    op.alter_column(
+        "active_tokens", "expiration", nullable=False, new_column_name="created"
+    )
```

Comment:
Wasn't this script automatically generated?  @dbutenhof, are you suggesting that we should edit these scripts after generating them?

Reply:
The script is generated, but usually edited. And I'd be surprised (though it wouldn't be the first time) if Alembic took it upon itself to decide that *renaming* a column was an appropriate upgrade when one column name disappears and another appears! In this case that works with existing data (simply pre-expiring all tokens), but in general it could be a really bad idea...

Reply:
I agree:  if we're entitled to edit this script, then we should be removing and creating (i.e, wiping) these columns instead of renaming them.

Reply:
I chose to use alter column instead of drop and create.  It seems silly to do anything about the table data since no token will be "created" in the future, and the first login will take care of the expirations.